docker-compose.yml

```diff
@@ -121,6 +123,9 @@ services:
     environment:
       << : *govuk-app
       GOVUK_APP_NAME: rummager-worker
+      RACK_ENV: production
+      REDIS_URL: redis://redis
+      REDIS_URL: redis://redis
```

Comment:
Don't think we need both of these 😄

Reply:
Oops thanks! - Fixed in https://github.com/alphagov/publishing-e2e-tests/pull/407/commits/2874cac6f436e7d6c2df20dcd9db7b8004790863